src/OpenLoco/src/Windows/EditKeyboardShortcut.cpp

```diff
@@ -18,17 +20,16 @@ using namespace OpenLoco::Input;
 
 namespace OpenLoco::Ui::Windows::EditKeyboardShortcut
 {
-    static constexpr Ui::Size kWindowSize = { 280, 72 };
+    static constexpr Ui::Size32 kWindowSize = { 280, 72 };
 
     static loco_global<uint8_t, 0x011364A4> _editingShortcutIndex;
 
-    static constexpr Widget _widgets[] = {
-        makeWidget({ 0, 0 }, kWindowSize, WidgetType::frame, WindowColour::primary),                                                                     // 0,
+    static constexpr auto _widgets = makeWidgets(
+        Widgets::Frame({ 0, 0 }, kWindowSize, WindowColour::primary),                                                                                    // 0,
         makeWidget({ 1, 1 }, { kWindowSize.width - 2, 13 }, WidgetType::caption_25, WindowColour::primary, StringIds::change_keyboard_shortcut),         // 1,
         makeWidget({ 265, 2 }, { 13, 13 }, WidgetType::buttonWithImage, WindowColour::primary, ImageIds::close_button, StringIds::tooltip_close_window), // 2,
-        makeWidget({ 0, 15 }, { kWindowSize.width, 57 }, WidgetType::panel, WindowColour::secondary),                                                    // 3,
-        widgetEnd(),
-    };
+        Widgets::Panel({ 0, 15 }, { kWindowSize.width, 57 }, WindowColour::secondary)                                                                    // 3,
+    );
```

Comment:
Perhaps we should just drop these trailing comments.

Reply:
Done.